cvat/apps/engine/utils.py

```diff
@@ -105,4 +108,33 @@ def parse_specific_attributes(specific_attributes):
     return {
         item.split('=')[0].strip(): item.split('=')[1].strip()
             for item in specific_attributes.split('&')
-    } if specific_attributes else dict()
\ No newline at end of file
+    } if specific_attributes else dict()
+
+class SortingMethods(str, Enum):
+    DEFAULT = 'DEFAULT'
```

Comment:
The same comments as for the client:
- DEFAULT, NATIVE, CUSTOM aren't meaningful
- I would recommend to have low case for strings (e.g. 'random')
- Probably reversed can be a flag. I will vote to remove the method for now
- Class name should in SortingMethod